subprojects/docs/src/docs/release/notes.md

```diff
@@ -32,6 +32,23 @@ details of 2
 
 ## n
 -->
+## Security 
+
+### Removed debug logging for environment variables
+
+Gradle allows you to start processes in various ways, for example via `Project.exec`, `Project.javaexec`, running tests or starting a Gradle daemon.
```

Comment:
I think the key part of setting the context here is not how you can start processes in Gradle but the risk of leaking credentials via debug logging.

I propose the following introductory paragraph:
"Debug level logging may expose sensitive information in the build log output, for example in CI server logs. For this reason, Gradle displays a prominent warning when using debug level logging since version 6.4. One example of this risk is leaking secret values such as credentials stored in environmental variables. "

Reply:
I used your suggestion.